drivers/source/AnalogIn.cpp

```diff
@@ -23,14 +23,14 @@ namespace mbed {
 
 SingletonPtr<PlatformMutex> AnalogIn::_mutex;
 
-AnalogIn::AnalogIn(PinName pin)
+AnalogIn::AnalogIn(PinName pin) : vref(MBED_CONF_TARGET_DEFAULT_ADC_VREF)
```

Comment:
Can we have this value as an extra argument to the constructor ?
```suggestion
AnalogIn::AnalogIn(PinName pin, vref = MBED_CONF_TARGET_DEFAULT_ADC_VREF) : _vref(vref)
```

Reply:
Done